docs/index.md

```diff
@@ -299,7 +299,14 @@ Open a port forward to a Pod as a background task/thread.
 from kr8s.objects import Pod
 
 pod = Pod.get("my-pod")
-pf = pod.portforward(remote_port=1234, local_port=5678)
+# Listen on port 5678 on 127.0.0.1, forwarding to 5000 in the pod
+pf = pod.portforward(remote_port=5000, local_port=5678)
+
+# Listen on port 8888 on all addresses, forwarding to 5000 in the pod
+pf = pod.portforward(remote_port=5000, local_port=8888, address=["0.0.0.0"])
+  
+# Listen on port 8888 on localhost and selected IP, forwarding to 5000 in the pod
+pf = pod.portforward(port=5000, local_port=8888, address=["127.0.0.1", "10.19.21.23"])
```

Comment:
Would you mind moving these example to the doc strings on the `portforward()` methods instead of the README. That way they will show up throughout the documentation. We also have an [examples section](https://github.com/kr8s-org/kr8s/blob/main/docs/examples/pod_operations.md) that would be nice to update.